templates/page/page.tpl.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @file
+ * Returns the HTML for a single Drupal page.
+ *
+ * Complete documentation for this file is available online.
+ * @see https://drupal.org/node/1728148
+ */
+?>
+
+<div id="main" class="main-content" role="main">
+
+  <header class="header" id="header" role="banner">
+
+    <section class="usa-banner">
+      <div class="usa-grid">
+        <div class="usa-banner-content" id="main-content">
+
+          <h1><?php print $site_name; ?></h1>
+
+          <?php if ($site_slogan): ?>
+            <h2 class="usa-font-lead"><?php print $site_slogan; ?></h2>
+          <?php else: ?>
```

Comment:
Because `$site_slogan` is [empty when the slogan has been disabled in theme settings](https://www.drupal.org/node/1728148), I suggest cutting lines 23 through 27 and setting a default value elsewhere. Otherwise if someone changes the theme settings to indicate that they don't want to display a slogan, they'll get one anyway, and it'll be one they didn't choose.